templates/dashboard/base.html

```diff
@@ -13,6 +13,7 @@
     {% block meta %}
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+      <meta name="theme-color" content="#21915a" />
```

Comment:
Some `{# comment #}` would be nice here, explaining that's for the mobile nav bar on Android mobiles

Reply:
It's not hack or unreadable code, and this meta tag is used since 2014, so I don't see any point in commenting it - it should be already known to anyone interested.

Reply:
I agree, it's not Android- or Chrome-specific either. We should consider other similar customizations: https://developers.google.com/web/fundamentals/design-and-ux/browser-customization/